src/components/NavBar/NavBar.tsx

```diff
@@ -23,11 +24,15 @@ export default function NavBar() {
   );
 
   return (
-    <nav className='page-fixed-width-root navbar'>
-      <ul className='navbar_list'>{links}</ul>
-    </nav>
+    <div className='row-flex margin-vertical-high'>
```

Comment:
ThemeToggle breaks positions here. nav element must be 600px wide and horizontally centered. It is being offset by ThemeToggle and is no longer centered